GOVERNANCE.md

```diff
@@ -0,0 +1,341 @@
+going to represent a substantive share of the stakeholders might not necessarily
+be represented in fair manner if one assumes that they use a (substantial)
+fraction of their mined coins to cover running costs.
+
+
+#### Proof of Work
+
+- allows miners to vote
+
+#### Proof of Stake
+
+- dash proposals gather ~25% masternode (~4500) votes
+- even if we allow stake holders and miners to contribute (via pos and pow
+  respectively) then there are still more groups of people: node operators,
+  who validate the chains, and end users, both of which have who have no good
+  way of voting. (Proof of Storage for node?)
+- non expert can appeal to »meta-expert agreement«, i.e. most experts go with
+  option one, therefore I should follow them
+
+
+#### Delegated Proof of Stake/Liquid democracy
+
+- direct democracy, flexible delegation, meta-delegation, and instant recall
+- policy area specific representation
+- the more expertise can be brought to bear on policy decisions, the more
+  likely is it that these decisions will advance the common good
+- »trustee model of representation, requires decision-makers to adhere to
+  deliberative norms, and balances liquid decision-making in legislatures
+  with an executive that reviews the formal feasibility of policies and
+  moderates package deals between proposals from different policy areas.«[3]
+
+#### Reputation
+
+- prevent people from directly influencing others/propaganda
+- (expertise =?more=) reputation (=?more= influence)
+
+#### Forking
+
+Having an inclusive approach when it comes to hard forks might ease a lot of the
+pains surrounding them. This could mean that nodes have the capability to track
+and interact with multiple forks at once. Or in general try not to frame it as
+an us vs. them issue but as a collaborative, we have to solve this together.
+
+Using votes to decide on forks could be seen as detrimental to this goal, since
+a vote has loser by design—if you are not voting for me, you're voting against
+me.
+
+
+#### Exit costs
+
+One of our goal is to make exit costs low, i.e. allow users to easily abandon
+undesirable forks. But it can be argued that if people have very low exit costs
+for a given system, then they incentive to vote according to their true beliefs,
+which should be the main goal for any sort of vote, is lowered and in turn would
+then render the vote pointless.
+
+And while the exit costs for a single voter can be quite low, the costs a fork
+imposes on a community can end up very high, especially when it results in two
+persistent chains. Both forks will have a vested interest in keeping the »original«
+name, code will have to be adjusted, wallets to be updated, nodes changed and
+users will definitely end up being confused.
+
+
+### Constitution
+
+- what is allowed to be decided on
+- rules on how to change rules are informed by expectations of what the outcomes
+  should/could be
+
+
+### Automatic protocol amendments
```

Comment:
The previous paragraph refers to "constitution". This paragraph has in title "protocol" though in its body refers again to "constitution", so I understand "protocol" here may be replaced with "constitution".

Reply:
No, these are two distinct issues. I'll push my latest changes in a bit